Parsedown.php

```diff
@@ -1453,8 +1458,19 @@ protected function elements(array $Elements)
         return $markup;
     }
 
+    #
+    # - [x] AND - [ ]
+    protected function checkbox(&$text,&$attributes)
+    {
+        if(strpos($text,'[x]')!==false||strpos($text,'[ ]')!==false){
+            $attributes = array("style"=>"list-style: none;");
```

Comment:
The GFM spec does not advise using an inline style here (see:  https://github.github.com/gfm/#task-list-items-extension-)